integration-testing/examples/user-project/build.gradle.kts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2016-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+plugins {
+    kotlin("multiplatform") version libs.versions.kotlinVersion.get()
+}
+
+repositories {
+    mavenCentral()
+    maven("https://maven.pkg.jetbrains.space/kotlin/p/kotlin/dev")
+    maven("../mpp-sample/build/.m2/")
```

Comment:
So, for now this solution works only for manual publishing of mpp-sample.

1. publishToMavenLocal atomicfu 0.23.1-SNAPSHOT

publish mpp-sample library to `mpp-sample/build/.m2/...`
2. `cd integration-testing/examples/mpp-sample/  gradle clean publish `
build user-project
3. `cd ../user-project/ gradle clean build `

And to write a test, I need to publish mpp-sample library to some known directory, so that I can provide a path to the artefact in user-project. 
What would be the best way to do that? 🤔